packages/react/src/components/Button/_button.scss

```diff
@@ -1,70 +1,72 @@
-@import '~carbon-components/scss/components/button/button';
-@import '../../globals/vars';
+@use '../../globals/theme';
+@use '../../globals/spacing';
+@use '../../globals/vars';
+@use "../../globals/colors";
 
-.#{$iot-prefix}--btn {
+.#{vars.$iot-prefix}--btn {
   justify-content: flex-start;
 
-  .#{$prefix}--loading {
-    margin-top: -$spacing-05;
-    margin-bottom: -$spacing-05;
-    margin-right: $spacing-03;
+  .#{vars.$prefix}--loading {
+    margin-top: -(spacing.$spacing-05);
+    margin-bottom: -(spacing.$spacing-05);
+    margin-right: spacing.$spacing-03;
   }
-  .#{$prefix}--loading__stroke {
-    stroke: $carbon--gray-50;
+  .#{vars.$prefix}--loading__stroke {
+    stroke: colors.$gray-50;
```

Comment:
`@carbon--` prefixes have been removed from all variables and are loaded from their appropriate modules.